docker/federation-test/run-tests.sh

```diff
@@ -5,6 +5,14 @@ pushd ../../server/
 cargo build
 popd
 
+pushd ../../ui
+yarn
+yarn build
+popd
```

Comment:
This isnt really needed because the test doesnt use the frontend at all.

Reply:
ah, didn't know that